ui/oopsyraidsy/damage_tracker.ts

```diff
@@ -159,6 +159,12 @@ export class DamageTracker {
       role: this.role,
       party: this.playerStateTracker.partyTracker,
       inCombat: this.inCombat,
+      isImmune: (targetId?: string) => {
+        const invulnIds = ['52', '199', '32A', '32B', '72C'];
```

Comment:
Can you leave a comment as to what these all are? Mostly so that if/when one changes in the future it's easier to come and edit this.